final/scripts/pca_final.py

```diff
@@ -73,30 +72,30 @@
     data_2d = data_2d - np.mean(data_2d, axis=1)[:, None]
     masked_data_2d = masked_data_2d - np.mean(masked_data_2d, axis=1)[:, None]
 
-    # PCA analysis on unmasked data: 
-    # Do SVD on the time by time matrix and get explained variance.
-    U, S, VT = npl.svd(data_2d.T.dot(data_2d))
-    exp_var = S / np.sum(S)
-    var_sums = np.cumsum(exp_var)
-
     # PCA analysis on MASKED data: 
     # Do SVD on the time by time matrix and get explained variance.
     U_masked, S_masked, VT_masked = npl.svd(masked_data_2d.T.dot(masked_data_2d))
     exp_var_masked = S_masked / np.sum(S_masked)
     var_sums_masked= np.cumsum(exp_var_masked)
-    masked_var_array[:,j] = exp_var_masked[:50] # Store the first 50 variance proportions.
+    masked_var_array[:,j] = exp_var_masked[:10] # Store the first 10 variance proportions.
```

Comment:
Originally, I was pulling the first 50 variances out for each subject's PCA because I wasn't sure what I'd be doing with them and 50 seemed like a decently large number without going crazy. All of my final plots work with only the first 10, though, so it seemed silly to have the extra 40 floating around when I never ended up doing anything with them.